src/main/java/hudson/plugins/sectioned_view/TextSection.java

```diff
@@ -87,10 +87,10 @@ public String getDisplayName() {
      */
     public enum Style {
         NONE("None", ""),
-        NOTE("Note", "sectioned-view-note"),
-        INFO("Info", "sectioned-view-info"),
-        WARN("Warning", "sectioned-view-warning"),
-        TIP("Tip", "sectioned-view-tip");
+        NOTE("Note", "jenkins-alert-warning"),
```

Comment:
was NOTE → jenkins-alert-warning intentional? Seems like NOTE (neutral) would better map to jenkins-alert-info, with WARNING using jenkins-alert-warning.